api/ServiceWorkerGlobalScope.json

```diff
@@ -1383,6 +1571,53 @@
           }
         }
       },
+      "serviceWorker": {
+        "__compat": {
+          "support": {
+            "chrome": {
+              "version_added": "79"
+            },
+            "chrome_android": {
+              "version_added": "79"
+            },
+            "edge": {
+              "version_added": "79"
+            },
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "66"
+            },
+            "opera_android": {
+              "version_added": "57"
+            },
+            "safari": {
+              "version_added": false
```

Comment:
No support for `self.serviceWorker` in Safari is surprising, but the test is as simple as it can be so I trust the results...